### ISSUE TITLE
Lusuco isn't on the map

### ISSUE THREAD

**Issue by aryes35** [Wynncraft/Issues#5262]
I checked the world map on the site and Lusuco isn't on there, it would kind of be near the Tower of Ascension, I tried refreshing the page and checking to see if all my markers were on and nothing, I tried to find it by looking for the potion selling indicator on the map but that's not showing up either...

**Comment by Moe-Ronickah** [comment]
The map is being updated.  When it will be finished and rolled out, we don't know.

**Comment by DRedDogPE** [comment]
Even if it were on the map you would not be able to see it very well. 

https://map.wynncraft.com/#/-171/64/-399/-2/0/0

**Comment by Moe-Ronickah** [comment]
I'm going to close this issue for one reason: the game and website admins will never put everything onto the map as it would become horribly cluttered.  What they choose to put or not put on the map has nothing to do with most suggestions, actually, as they base those decisions on other factors.  I will pass this along to the new website admin, and if he changes it, great.  If he doesn't, it's because the admins have some reason for not doing it.

Long story short: I'll pass it along, and thanks for pointing it out, but we have no idea if it will get changed, nor when the map will get updated.
